client/public/index.html

```diff
@@ -2,13 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link rel="icon" href="2011_Logo_white.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#E4E4E7" />
+    <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="icon" href="2011_Logo_white.png" />
     <link rel="apple-touch-icon" href="2011_Logo_white.png" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
```

Comment:
I don't think this file should've changed, I changed the `theme-color` to `#E4E4E7` a couple months ago